dev.compose.yaml

```diff
@@ -0,0 +1,32 @@
+version: "3.9"
+
+services:
+  postgres:
+    container_name: "snailycad-bot-postgres"
+    image: postgres
+    command: -p ${DB_PORT:-5554}
+    env_file:
+      - .env
+    ports:
+      - "${DB_PORT:-5554}:${DB_PORT:-5554}"
+    volumes:
+      - snailybot-dev-db-data:/var/lib/postgresql/data
+
+  api:
+    container_name: "snaily-cad-bot"
+    image: snaily-cad-bot:dev
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+      target: development
+    volumes:
+      - .:/srv
+    env_file:
+      - .env
+    environment:
+      - NODE_ENV=development
+    depends_on:
+      - postgres
+
+volumes:
+  snailybot-dev-db-data:
```

Comment:
This volume will be removed when the Docker container is removed, this can lead to unwanted things such as dataloss. Please use a local volume like before.

Reply:
If we keep the local mount folder, the build is failed on Linux hosts, with this error

`failed to solve: rpc error: code = Unknown desc = error from sender: open <project_path>/snailycad-bot/dev-data: permission denied`